config/prod/xdoan-clarisse-htan-s3.yaml

```diff
@@ -0,0 +1,43 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: remote/s3-bucket.yaml
+stack_name: xdoan-clarisse-htan-s3
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "HTAN"
+  # The resource owner
+  OwnerEmail: "xengie.doan@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) Synapse username (default ""), required if AllowWriteBucket=true
+  SynapseUserName: 'clarisselau'
+  # (Optional) Allow accounts, groups, and users to access bucket. (default is no access).
+  GrantAccess:
+    - 'arn:aws:iam::426577889437:user/clarisse.lau'   # Required ARN for a synapse bucket
```

Comment:
don't you want to give yourself access as well?

Reply:
Just added my arn and the root arn, thanks for the catch!